cmd/http-tracer.go

```diff
@@ -179,7 +179,8 @@ func Trace(f http.HandlerFunc, logBody bool, w http.ResponseWriter, r *http.Requ
 	if bodyContents != nil {
 		rs.Body = bodyContents
 	}
-	if !logBody {
+
+	if rs.StatusCode <= 400 || rs.StatusCode >= 599 && !logBody {
```

Comment:
```suggestion
	if rs.StatusCode < 400 && !logBody {
```

I think this is the intention here. HTTP Status codes below 400 are not errors, and those >= 400 are errors. There's no need to make an upper limit like 599 as it not needed, and indeed there was an operator precedence problem in the earlier code.

Reply:
Thanks for catching the operator precedence issue!